config.json

```diff
@@ -999,7 +999,7 @@
         "auto_landing_warning_distance": 350.0,
         "auto_pilot_compensate_for_interdiction": false,
         "auto_pilot_no_enemies_distance_multiplier": 4.0,
-        "auto_pilot_planet_radius_percent": 0.2,
+        "auto_pilot_planet_radius_percent": 0.5,
```

Comment:
This change causes the ship to drop out of autopilot at the correct distance from the target planet, just before docking distance. Although perhaps 0.49 would be an even better setting.